modules/api/php/endpoints/candidates.class.inc

```diff
@@ -241,7 +241,7 @@ class Candidates extends Endpoint implements \LORIS\Middleware\ETagCalculator
                 $pscid,
                 $project->getId()
             );
-        } catch (\LorisException | \InvalidArgumentException $e) {
+        } catch (\Exception | \LorisException | \InvalidArgumentException $e) {
```

Comment:
When is a `\LorisException` or an `\InvalidArgumentException` not an `\Exception`?

Reply:
When PSCID is not provided and generation is user

Reply:
You mean that by adding \Exception I should remove the other two because they are subclasses?